src/utils/figmaTheme/getFigmaTheme/getFigmaTheme.js

```diff
@@ -42,7 +42,9 @@ export const getFigmaTheme = (files, theme) => {
   }
 
   const sourcePartialsToLoad = tokenSetOrder.filter(
-    (themeName) => themeData.selectedTokenSets[themeName] !== 'disabled'
+    (themeName) =>
+      Boolean(themeData.selectedTokenSets[themeName]) &&
+      themeData.selectedTokenSets[themeName] !== 'disabled'
```

Comment:
Can `themeName` or `themeData` ever be `null` or `undefined`?

NITTTppiiickkkkkk!

Reply:
They won't be null. The item from the selected tokensets can be null/undefined. thats what we're filtering out right now

Reply:
Ofc they won't, what was I thinking. Drory always crosses his t's and dots his i's with fresh pots.